src/main/java/com/naveenautomation/Pages/CamerasPage.java

```diff
@@ -0,0 +1,40 @@
+package com.naveenautomation.Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import com.naveenautomation.Browsers.ProxyDriver;
+
+public class CamerasPage extends Page {
+
+	private static final String PAGE_URL = "product/category&path=33";
```

Comment:
this URL is not correct, it should be just product/

Reply:
URL changed as recommended.